articles/service-fabric/service-fabric-understand-and-troubleshoot-with-system-health-reports.md

```diff
@@ -53,6 +53,16 @@ The report specifies the global-lease timeout as the time-to-live (TTL). The rep
 * **Property**: Starts with **Neighborhood** and includes node information.
 * **Next steps**: Investigate why the neighborhood is lost, for example, check the communication between cluster nodes.
 
+### Rebuild
```

Comment:
What is rebuild? customers have never heard the term. When can it happen, is it bad?